client/app/screen_two/page.tsx

```diff
@@ -1,22 +1,21 @@
-"use client";
+  "use client";
 
-/**
- * @author @abhi-arya1
- * @function TelemetryPage
- * @fileoverview Coming once all "TODO:" are complete
- */
+  /**
+   * @author @abhi-arya1 @ivanvuong
+   * @function TelemetryPage
+   * @fileoverview Coming once all "TODO:" are complete
+   */
 
-import Map from "@/components/ui/map";
+  import { CameraFeed, TempYoutubeVideo } from "@/components/hmd_link/camera_feed";
+  import Map from "@/components/ui/map";
 
+  function TelemetryPage() {
+    return (
+      <div className="h-full flex flex-row gap-x-4 items-center justify-center">
+          <CameraFeed />
+          <Map />
```

Comment:
there's an emergent issue with the map (yet again). i'll be opening an issue. We'll merge this PR with that `<Map />` line commented out as such: `{ /* <Map /> */ }`.